elastic/beats
# [MongoDB] Bug-fix change for authenticating multiple host

- Bug

## What does this PR do?

This PR fixes the multiple host connection in mongodb module.  

## Why is it important?



## Checklist





## How to test this PR locally

Run mongodb in multiple instance.
Enable mongodb module and update mongodb.yml with host and port details.
Provided the connection string in the below format: 
`hosts: ["mongodb://beats:pass@localhost:27021","localhost:27022","localhost:27023"]`
Start metricbeat and metrics should start flowing through ES.

Attached is the sample event outcome through manual test: 

## Related issues

- Closes #32188
